shoop/core/models/_shops.py

```diff
@@ -46,6 +46,8 @@ class Shop(ChangeProtected, TranslatableShoopModel):
     prices_include_tax = models.BooleanField(default=True, verbose_name=_("prices include tax"))
     logo = FilerImageField(verbose_name=_("logo"), blank=True, null=True, on_delete=models.SET_NULL)
     maintenance_mode = models.BooleanField(verbose_name=_("maintenance mode"), default=False)
+    contact_address = models.ForeignKey(
```

Comment:
As you said, it's not a problem. And the address being immutable doesn't even help, since you can still change the contact_address field in Shop to point a different immutable address. :)